src/elements/content-explorer/ContentExplorer.js

```diff
@@ -761,12 +795,16 @@ class ContentExplorer extends Component<Props, State> {
         }
 
         this.unselect();
-        item.selected = true;
 
-        const focusedRow = items.findIndex((i: BoxItem) => i.id === item.id);
-        this.setState({ focusedRow, selected: item }, () => {
-            onSelect(cloneDeep([item]));
-            callback(item);
+        const selectedItem: BoxItem = { ...item };
+        selectedItem.selected = true;
```

Comment:
can do this in one line: `const selectedItem: BoxItem = { ...item, selected: true };`